src/main/java/org/broadinstitute/hellbender/engine/spark/SparkCommandLineProgram.java

```diff
@@ -0,0 +1,61 @@
+package org.broadinstitute.hellbender.engine.spark;
+
+import org.apache.spark.SparkConf;
+import org.apache.spark.api.java.JavaSparkContext;
+import org.broadinstitute.hellbender.cmdline.Argument;
+import org.broadinstitute.hellbender.cmdline.CommandLineProgram;
+
+import java.io.Serializable;
+
+
+public abstract class SparkCommandLineProgram extends CommandLineProgram implements Serializable {
+    private static final long serialVersionUID = 1l;
+
+    @Argument(doc = "API Key for google cloud authentication",
+            shortName = "apiKey", fullName = "apiKey", optional=true, mutex={"client_secret"})
```

Comment:
Remove the mutex, since spark `CommandLinePrograms` don't have a `client_secret` argument.